docs/source/getting_started.rst

```diff
@@ -1,66 +1,29 @@
-Getting Started
-===============
+Getting Started - Install
+=========================
 
 .. toctree::
    :maxdepth: 1
    :hidden:
 
    prereqs
    install
-   test_network
+   sdk_chaincode
 
-Before we begin, if you haven't already done so, you may wish to check that
-you have all the :doc:`prereqs` installed on the platform(s)
-on which you'll be developing blockchain applications and/or operating
-Hyperledger Fabric.
 
-Once you have the prerequisites installed, you are ready to download and
-install HyperLedger Fabric. While we work on developing real installers for the
-Fabric binaries, we provide a script that will :doc:`install` to your system.
-The script also will download the Docker images to your local registry.
+The Fabric application stack has 5 layers:
```

Comment:
Fabric is an "application stack"? Are we comfortable with that term?

Also at a minimum this should be "five".

Reply:
Switched to "five". Open to suggestions on application stack.